salt/modules/iptables.py

```diff
@@ -390,7 +390,7 @@ def build_rule(table=None, chain=None, command=None, position='', full=None, fam
             rule += '--{0} {1} '.format(item, kwargs[item])
 
     for item in after_jump:
```

Comment:
You need to delete this loop, otherwise you'll be adding all the items after the jump `len(after_jump)` times.